assets/agenda/utils.js

```diff
@@ -405,6 +405,25 @@ export function getInternalNotesFromCoverages(item) {
     return internalNotes;
 }
 
+/**
+ * Get list of subjects
+ *
+ * @param {Object} item
+ * @return {Array}
+ */
+export function getSubjects(item) {
```

Comment:
shouldn't this check for duplicates at some point?

Reply:
True. I'll add that check.